notebooks/02_Data_Sync.ipynb

```diff
@@ -53,20 +53,21 @@
     "  inserts, see\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    env_vars = os.environ
</pre><p>Does this check still work with the current config setup?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/spyglass/pull/735/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>You're right, this is no longer mandatory.  I'll take it out</p>